apache-couchbase-mixin/dashboards/couchbase-cluster-overview.libsonnet

```diff
@@ -6,13 +6,12 @@ local prometheus = grafana.prometheus;
 
 local dashboardUid = 'apache-couchbase-cluster-overview';
 
-local promDatasourceName = 'prometheus_datasource';
+local promDatasourceName = 'datasource';
```

Comment:
Hi - is this done only to pass the linting rule?

Reply:
Yes. I realize I could also add a linting rule to overlook this, but since 'Datasource' is the name of the variable in the dashboard, I figured this fit better.